### PR TITLE
Added descriptions to each proxy service in perf test.

### PR DESCRIPTION
**Pull request by Maninda**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: distribution/scripts/jmeter/performance-test-scenarios.sh [hunk 1]
```diff
@@ -48,6 +49,7 @@ declare -A test_scenario0=(
 declare -A test_scenario1=(
     [name]="CBRProxy"
     [display_name]="CBRProxy"
+    [description]="Routing message by reading a text field in message body"
```
Comment (isudana) [inline distribution/scripts/jmeter/performance-test-scenarios.sh]: Shall we change to "Routing the message based on the content of the message body"

File: distribution/scripts/jmeter/performance-test-scenarios.sh [hunk 2]
```diff
@@ -57,6 +59,7 @@ declare -A test_scenario1=(
 declare -A test_scenario2=(
     [name]="CBRSOAPHeaderProxy"
     [display_name]="CBRSOAPHeaderProxy"
+    [description]="Routing message by reading a SOAP header in message payload"
```
Comment (isudana) [inline distribution/scripts/jmeter/performance-test-scenarios.sh]: Routing the message based on a SOAP header in the message payload

File: distribution/scripts/jmeter/performance-test-scenarios.sh [hunk 3]
```diff
@@ -66,6 +69,7 @@ declare -A test_scenario2=(
 declare -A test_scenario3=(
     [name]="CBRTransportHeaderProxy"
     [display_name]="CBRTransportHeaderProxy"
+    [description]="Reading transport header and route to backend"
```
Comment (isudana) [inline distribution/scripts/jmeter/performance-test-scenarios.sh]: Routing the message based on an HTTP header in the message

File: distribution/scripts/jmeter/performance-test-scenarios.sh (hunk 2 above)
Comment (Maninda) [reply distribution/scripts/jmeter/performance-test-scenarios.sh]: Changed

File: distribution/scripts/jmeter/performance-test-scenarios.sh (hunk 1 above)
Comment (Maninda) [reply distribution/scripts/jmeter/performance-test-scenarios.sh]: Changed

File: distribution/scripts/jmeter/performance-test-scenarios.sh (hunk 3 above)
Comment (Maninda) [reply distribution/scripts/jmeter/performance-test-scenarios.sh]: Changed

**Review by isudana — APPROVED** [review]

**Pull request merged**